app/src/main.cpp

```diff
@@ -429,15 +361,7 @@ namespace scitt
             "Failed to get host time: {}", ccf::api_result_to_str(result)));
         }
 
-        try
-        {
-          post_entry_common(ctx, host_time, body);
-        }
-        catch (const did::web::AsyncResolutionNeeded& e)
-        {
-          start_asynchronous_registration(ctx, host_time, e.did, body);
```

Comment:
I think we should also be able to remove all the [operations endpoints](https://github.com/microsoft/scitt-ccf-ledger/blob/4f4eb5310afb52c17e7fd4832c3d7d3bdbfd2e1d/app/src/operations_endpoints.h#L525), since they were primarily used to check the status of a post entry operation when there was a did:web resolution.

Reply:
I suppose I still need to port some logic from the `operations_endpoints` into the post_entry endpoint since the response will be empty otherwise, right ?
e.g. `operation_locally_committed_func`

Reply:
Operations endpoints are standard handlers in azure to deal with async ops, sdk depends on them at the moment, if you were to remove those then the response needs to match the expectations in SCRAPI SCITT API proposals I think, @achamayou could suggest here
scrapi spec draft https://github.com/ietf-wg-scitt/draft-ietf-scitt-scrapi/blob/main/draft-ietf-scitt-scrapi.md#register-signed-statement

Reply:
From Azure API perspective post must return the location of the created resource as a URL https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#http-return-codes just like in the case of SCRAPI, it is just SCRAPI suggests the response be in `Payload (in CBOR diagnostic notation)` which is kind of weird given they also use JSON for errors.

Reply:
Interestingly, Darrel Miller has written a review of the SCITT architecture containing detailed views on errors and long running operations: https://datatracker.ietf.org/doc/review-ietf-scitt-architecture-01-httpdir-early-miller-2023-04-24/

For now I think we:

- Keep the handling for async commit as is
- Remove the did resolution handling

Once we've moved over to the new receipt format, we can do a pass over the exact details of the API and errors.

Reply:
I've restored the operation endpoints, kept the same data structures and removed DID:Web async resolution